Objects/Converters/ConverterRhinoGh/ConverterRhinoGhShared/ConverterRhinoGh.Mappings.cs

```diff
@@ -60,9 +60,24 @@ private Base MappingToSpeckle(string mapping, RhinoObject @object, List<string>
 
         //NOTE: this works for BOTH the Wall.cs class and RevitWall.cs class etc :)
         case Wall o:
-          var extrusion = (RH.Extrusion)@object.Geometry;
-          var bottomCrv = extrusion.Profile3d(new RH.ComponentIndex(RH.ComponentIndexType.ExtrusionBottomProfile, 0));
-          var topCrv = extrusion.Profile3d(new RH.ComponentIndex(RH.ComponentIndexType.ExtrusionTopProfile, 0));
+          var wallBrep = @object.Geometry is RH.Brep wallB ? wallB : ((RH.Extrusion)@object.Geometry)?.ToBrep();
```

Comment:
Working with BREP instead is a nice move, but I'm guessing that this is intended for "Single Surface BREPs", not for complex multi face ones.

Maybe we should throw a meaningful message if the BREP has more than 2 surfaces? Or are we doing this somewhere else in the mapping tool?

Reply:
Ok then we can go ahead, as any other weird brep will fall under "invalid" because it will have more than 2 edges for sure.